.azure/gpu-tests-pytorch.yml

```diff
@@ -106,29 +101,21 @@ jobs:
         for fpath in `ls requirements/**/*.txt`; do \
           python ./requirements/pytorch/adjust-versions.py $fpath ${PYTORCH_VERSION}; \
         done
+        # prune packages with installation issues
+        pip install -q -r .actions/requirements.txt
+        python .actions/assistant.py requirements_prune_pkgs \
```

Comment:
these shall not be listed among strategies as they introduce cyclic depedency

Reply:
They are not listed as strategies in setup or anything like that